JSON/Simon's Ultimate Showdown.json

```diff
@@ -15,5 +15,6 @@
     "Status": "Considered"
   },
   "SteamID": "2158339043",
+  "TutorialVideoUrl": "https://youtu.be/tnBSg7k62yo",
```

Comment:
No Youtube link in form of `youtu.be` Please use the link as shown on the address bar (without any extra such as `&t` of course).